pygsti/evotypes/chp/statereps.py

```diff
@@ -87,25 +77,3 @@ def reps_have_changed(self):
     def actionable_staterep(self):
         state_rep = self.state_rep.actionable_staterep()
         return self.op_rep.acton(state_rep)
-
-#REMOVE
-#    def chp_ops(self, seed_or_state=None):
-#        return self.state_rep.chp_ops(seed_or_state=seed_or_state) \
-#            + self.op_rep.chp_ops(seed_or_state=seed_or_state)
-
-# TODO: Untested, only support computational and composed for now
-#class StateRepTensorProduct(StateRep):
```

Comment:
For Stefan to check out

Reply:
@sserita did you look into this? If not, could you open a GitHub issue articulating the question that needs investigation?